doc/classes/Particles2D.xml

```diff
@@ -295,7 +295,7 @@
 			If [code]true[/code] particles are being emitted. Default value: [code]true[/code].
 		</member>
 		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
-			Time ratio between each emission. If [code]0[/code] particles are emitted continuously. If [code]1[/code] all particles are emitted simultaneously. Default value: [code]0[/code].
+			How rapidly particles in an emission cycle are emitted. If more than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
```

Comment:
Change "more than" to "greater than"

Reply:
Done. Sorry for having so much to fix. :p